From 6ae6dea480e62033b53f7bccbe446a817473c942 Mon Sep 17 00:00:00 2001
From: lemoer <git@irrelefant.net>
Date: Mon, 23 Apr 2018 01:00:37 +0200
Subject: [PATCH] Revert "Revert "generic: batman-adv: disable multicast
 optimizations for now""

This reverts commit 23a8947628aaed5ddb1e83aa938f7cbf69fc2fd8.
---
 targets/generic | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/targets/generic b/targets/generic
index 37c16be0..eb821e31 100644
--- a/targets/generic
+++ b/targets/generic
@@ -21,6 +21,8 @@ config '# CONFIG_PACKAGE_usbip is not set' # fails to build
 config 'CONFIG_BUSYBOX_CUSTOM=y'
 config '# CONFIG_BUSYBOX_CONFIG_FEATURE_PREFER_IPV4_ADDRESS is not set'
 
+config '# CONFIG_KMOD_BATMAN_ADV_MCAST is not set'
+
 config 'CONFIG_PACKAGE_ATH_DEBUG=y'
 
 try_config 'CONFIG_TARGET_MULTI_PROFILE=y'
-- 
2.16.2

